smelt.lic

```diff
@@ -25,80 +17,89 @@ class Smelt
 
     args = parse_args(arg_definitions)
 
-    get_crafting_item(@rod, @bag, @bag_items, @belt)
+    @settings = get_settings
+    @bag = @settings.crafting_container
+    @bag_items = @settings.crafting_items_in_container
+    @belt = @settings.forging_belt
+    @rod = @settings.forging_tools.find { |item| /rod/ =~ item }
+    @adjustable_tongs = @settings.adjustable_tongs
+
+    get(@rod)
     if args.refine
-      refine
+      get('flux')
+      type("pour", "flux", "in crucible", "flux", true)
```

Comment:
How does the `refine` branch work? I see it gets the flux, sets the instance vars, but then where does it actually do the work?

Reply:
ah, it doesn't, missed that.

Reply:
https://github.com/rpherbig/dr-scripts/pull/5768/commits/4ad68643180b44086ac4a15ab2757be88b0da7ca has the change for that